media/doc/WINESYNC.txt

```diff
@@ -263,8 +263,7 @@ check Wine current sources first as it may already be fixed.
 sdk/lib/3rdparty/strmbase           # Synced to WineStaging-3.3
 
 sdk/lib/rtl/actctx.c                # Partly synced with WineStaging-1.9.16
-sdk/lib/rtl/timerqueue.c            # Partly synced with WineStaging-1.7.55
```

Comment:
To confirm, what is left of these file is ROS code, not WINESYNC?

Reply:
yeah it’s an export we have that wine doesn’t